docs/user-guide/quickstart.md

```diff
@@ -308,7 +308,7 @@ shell in the client container, confirm that we can access the `s2` service runni
 in namespace `service-2`.
 
 ```bash
-kubectl run s2 -n service-2 --image=nginx --restart=Never --expose --port 80
+kubectl run s2 -n service-2 --image=nginxinc/nginx-unprivileged --restart=Never --expose --port 8080
```

Comment:
Oh, I think I'd use the standard port 80 rather than make this command so much longer. Do you mind making that change (and changing the other 8080s to 80s, of course)?